### PR TITLE
[ADMIN] Vaisala PR into Test

### PR DESCRIPTION
**Pull request by seanmcilroy29**
Vaisala branch
- [x] Validated with no errors
- [x] Validated well-formed

### CODE REVIEW

**Comment by seanmcilroy29** [comment]
PR updated with OMA IPR Policy objects

 - [x] Validated with no errors
 - [x] Validated well-formed

File: 3338-1_1.xml [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+				<Operations>RW</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>String</Type>
+				<RangeEnumeration></RangeEnumeration>
+				<Units></Units>
+				<Description>The application type of the sensor or actuator as a string depending on the use case.</Description>
+			</Item>
+			<Item ID="5518">
+				<Name>Timestamp</Name>
```
Comment (akeranen) [inline 3338-1_1.xml:101]: Is timestamp in buzzer intentional? "when measurement was performed" seems a bit odd here.

Comment (jpradocueva) [reply 3338-1_1.xml:101]: @akeranen your comments are in a branch that was previously agreed by IPSO WG. 
The plan was to update this branch with the same changes as in Common. At that time there will be an option to update this content.

Comment (akeranen) [reply 3338-1_1.xml:101]: Sounds good. Where should we make a note of that so that it's not forgotten?

Comment (jpradocueva) [reply 3338-1_1.xml:101]: @akeranen @dirtyha    At the last IPSO call, it was agreed:

Before asking Harri  (Vaisala) to run his tool again to create versions 1.1 of IPSO objects Matt will provide a proposal on how to release Objects in this registry: block or single releases.
Once this is clear the staff will prepare the repository for that particular of release type. Once the repository is ready, we can ask to Harri to run his tool and create a new set of IPSO Objects. If we try to do it know we will have near twice the amount work when preparing the release of objects in this repository.

This release that I am referring to is that currently, all the versions of an object is in the same branch. But the group wants to remove versions from the filename, this implies that there will be a single version of an object in the same branch. Previous versions will be stored in  the "release branches".

**Comment by jpradocueva** [comment]
This PR is closed as it will be superseded by a new PR that will create version 1.1 of the IPSO Obects.

**Pull request closed without merging**